### ISSUE TITLE
Karma/Jasmine Lightning Talk

### ISSUE THREAD

**Issue by Seberius** [renodevelopers/meetups#40]
I have started using Karma for code testing on a project and can give a short presentation.  I would like cover what it is, its features, an example code test, a preprocessor with Google Closure example, and integration with IntelliJ IDEA.  I would estimate it to take ~10 minutes.

**Comment by w1zeman1p** [comment]
Did this already happen?

**Comment by Seberius** [comment]
No, this one never occurred.  It might make a good back up topic.
